src/Forms/FieldList.php

```diff
@@ -410,12 +385,12 @@ public function findTab($tabName)
      * @param string $tabName The tab to return, in the form "Tab.Subtab.Subsubtab".
      *   Caution: Does not recursively create TabSet instances, you need to make sure everything
      *   up until the last tab in the chain exists.
-     * @param string $title Natural language title of the tab. If {@link $tabName} is passed in dot notation,
+     * @param string|null $title Natural language title of the tab. If {@link $tabName} is passed in dot notation,
      *   the title parameter will only apply to the innermost referenced tab.
      *   The title is only changed if the tab doesn't exist already.
      * @return Tab The found or newly created Tab instance
```

Comment:
```suggestion
```
That doesn't provide any information that isn't obvious from the method signature.